beta-src/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "homepage": "beta",
+  "proxy": "http://localhost:8895",
```

Comment:
Are we able to put the port value into an .env file? Other devs are using other ports.

Reply:
@bryancodazen it would be great if you or someone from codazen could finish up this PR, since the remaining error is buried deep in the map code.

Reply:
Actually I have an idea of how to fix the remaining error, let me take a look and get back to you